README.md

```diff
@@ -64,6 +65,11 @@ To configure the zoom and pan plugin, you can simply add new config options to y
 			x: null,
 			y: null
 		},
+
+		// Speed of zoom via mouse wheel
+		// (percentage of zoom on a wheel event)
+		speed: 10,
```

Comment:
I probably would have gone with `0.1` since it's a percentage

Reply:
Yeah I wasn't sure about what was the most intuitive... I changed that.